docs/topics/proc_deploying-the-circuit-breaker-booster-using-the-oc-cli-client.adoc

```diff
@@ -9,11 +9,15 @@
 
 .Prerequisites
 
-* The booster application created using the {launcher} application.
+ifndef::parameter-openshiftlocal[]
```

Comment:
I suggest removing this `ifdef` and instead setting `:parameter-deployment:`, which will be set to

* `link:{link-launcher-oso}[{launcher-oso}]` when used in the OSO context, and
* `the {launcher} service` with OSL.

The first bullet point will not be conditionalized then, and will say:

`* The booster application created using {parameter-deployment}.`

Also please update the comment about parametres used.